aidial_adapter_bedrock/llm/model/claude/v3/adapter.py

```diff
@@ -93,6 +90,14 @@ async def chat(
                 if params.tool_config.is_tool
                 else ToolsMode.FUNCTION_EMULATION
             )
+        if tools_mode == ToolsMode.FUNCTION_EMULATION:
```

Comment:
We don't need to do this. If the request messages aren't compatible with the tools_mode, then it is an invalid request. We don't need to fix it.

In other words, when one sends tool-messages, but declares functions (or vice-versa), then it's an invalid request.

Reply:
What we should do is probably use a method like 
`def validate_tool_messages(tools_mode: ToolsMode, messages: List[BaseMessage | ToolMessage])`

Reply:
https://github.com/epam/ai-dial-adapter-bedrock/pull/116#discussion_r1647471183

I need this on `User response with function result` step

Because user works with API like with Functions, he uses Function message types

But I need to convert this messages to Tool message type, in order for model to process them correctly